aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTabMetricsCard.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ErrorBoundary from 'components/ErrorBoundary/ErrorBoundary';
+import Card from 'components/kit/Card/Card';
+import { Badge } from 'components/kit';
+
+import COLORS from 'config/colors/colors';
+
+import runDetailAppModel from 'services/models/runs/runDetailAppModel';
+
+import contextToString from 'utils/contextToString';
+
+function RunOverviewTabMetricsCard({
+  runData,
+  runHash,
+  runBatch,
+  type,
+}: {
+  runData: any;
+  runBatch: any;
+  type: 'metric' | 'systemMetric';
+  runHash: string;
+}) {
+  const [tableData, setTableData]: any = React.useState([]);
+
+  React.useEffect(() => {
+    if (!runBatch && !_.isNil(runData.runTraces)) {
+      const runsBatchRequestRef = runDetailAppModel.getRunMetricsBatch(
+        runData.runTraces.metric,
+        runHash,
+      );
+      runsBatchRequestRef.call();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [runData.runTraces, runHash]);
```

Comment:
This should be deleted, because RunOverviewSidebar already requested for runsBatches. Now In browser network tab you can see it requested twice.

Reply:
@arsengit sidebar is the part of overview maybe it would be better to take out this request from RunOverviewSidebar to RunOverviewTab and use it in nested components